### PR TITLE
Update investigate-malicious-email-that-was-delivered.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Link in the note and related topics for remediating malicious email delivered in office 365. This action was taken as the previous links were broken.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2642

File: microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ ms.custom: seo-marvel-apr2020
 [Office 365 Advanced Threat Protection](office-365-atp.md) enables you to investigate activities that put people in your organization at risk, and to take action to protect your organization. For example, if you are part of your organization's security team, you can find and investigate suspicious email messages that were delivered. You can do this by using [Threat Explorer (or real-time detections)](threat-explorer.md).
 
 > [!NOTE]
-> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/mtp/article-submission/remediate-malicious-email-delivered-office-365?view=o365-21vianet&branch=pr-en-us-4258).
+> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md]:
```suggestion
> To learn more about remediation, see [Remediate malicious email that was delivered in Office 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 please copy review. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Review by MSFTTracyP — CHANGES_REQUESTED** [review]
Please don't make the article more dry than it has to be. Leave the phrasing on line 29. I have no problem with the 269 edit. Thanks for that. I really appreciate your work on the links. But please retain the action verbs where we can sneak them in.

File: microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ ms.custom: seo-marvel-apr2020
 [Office 365 Advanced Threat Protection](office-365-atp.md) enables you to investigate activities that put people in your organization at risk, and to take action to protect your organization. For example, if you are part of your organization's security team, you can find and investigate suspicious email messages that were delivered. You can do this by using [Threat Explorer (or real-time detections)](threat-explorer.md).
 
 > [!NOTE]
-> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/mtp/article-submission/remediate-malicious-email-delivered-office-365?view=o365-21vianet&branch=pr-en-us-4258).
+> To learn more about remediation, see [Remediate malicious email that was delivered in Office 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md]:
```suggestion
> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review -as per @MSFTTracyP's request.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP The PR is ready to be merged. Kindly merge it. Thanks.

**Pull request merged**